layers/cmd/pipeline.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"io"
+
+	"github.com/cloudprivacylabs/opencypher/graph"
+)
+
```

Comment:
Make a new cobra cmd here to do `layers pipeline <file> <inputFile>`. Read the pipeline file from `file`, and call each step.